### PR TITLE
Bugfix pending unread

### LINKED ISSUE #74: Remove pending indicator when marking
When marking an outgoing message as unread, the pending indicator (3 dots) appears at the bottom of the message which was marked as unread. This may be a problem with notifying a message as pending, and this may have to change.

As per request, move the read reminder icon closer to the message bubble

image

# Demo Steps:
1. Mark an outgoing message as unread
2. Re-enter the conversation
3. Message looks normal, and icon is closer to the message bubble

# Acceptance tests

Testing Component | Inputs | Outputs | Status
------ | ------- | --------- | --------
Pending Indicator | Mark message as unread | Only read reminder should be set, no other indicators | PASS |
Read Reminder | Mark a message as unread | Read reminder icon should be on inside of message bubble | PASS |
Read Reminder | Mark message as unread, then tap message | Read reminder should disappear | PASS

Priority: 1/4
Risk: 1/4
Ideal hours: 2

### PR DESCRIPTION
**Pull request by claudds**
Closes #74 

Addressing the bug regarding the pending indicator popping up when an outgoing message was marked as unread. Fixed this by moving the indicator setting to a different conditional block.

Changed the icon to be more consistent with the menu icon. Also moved this icon to the inside of the message bubbles. Tapping message to remove icon is still functional.

### CODE REVIEW

File: res/layout/conversation_item_received.xml [hunk 1]
```diff
@@ -127,6 +127,22 @@
                     android:contentDescription="@string/conversation_item_sent__message_pinned_description"
                     tools:visibility="visible"/>
 
+                <ImageView
+                    android:id="@+id/read_reminder"
+                    android:layout_width="12dp"
+                    android:layout_height="11dp"
+                    android:src="@drawable/icon_notification"
```
Comment (jacobrs) [inline res/layout/conversation_item_received.xml:134]: does this mean the icon will be blue next to a bunch of grey icons?

File: res/layout/conversation_item_sent.xml [hunk 2]
```diff
@@ -189,6 +189,21 @@
                         android:contentDescription="@string/conversation_item_sent__message_pinned_description"
                         tools:visibility="visible"/>
 
+                <ImageView
+                    android:id="@+id/read_reminder"
+                    android:layout_width="12dp"
+                    android:layout_height="11dp"
+                    android:src="@drawable/icon_notification"
```
Comment (jacobrs) [inline res/layout/conversation_item_sent.xml:196]: same as above.

File: src/org/thoughtcrime/securesms/ConversationItem.java [hunk 3]
```diff
@@ -527,7 +529,7 @@ private void setFallbackStatusIcons() {
   }
 
   private void setUnreadStatusIcons(){
-    alertView.setUnread();
+    ;
```
Comment (jacobrs) [inline src/org/thoughtcrime/securesms/ConversationItem.java]: is this method required?

File: res/layout/conversation_item_received.xml (hunk 1 above)
Comment (claudds) [reply res/layout/conversation_item_received.xml:134]: It's actually a completely different Icon. I will remove the exclamation point image for this pr, but the image is now the Signal message bubble, which gets tinted the same grey as the rest of the icons

File: res/layout/conversation_item_sent.xml (hunk 2 above)
Comment (claudds) [reply res/layout/conversation_item_sent.xml:196]: see comment above

File: src/org/thoughtcrime/securesms/ConversationItem.java (hunk 3 above)
Comment (claudds) [reply src/org/thoughtcrime/securesms/ConversationItem.java]: will remove!

**Review by jacobrs — APPROVED** [review]

**Review by benrs — APPROVED** [review]
Looks good to me.

**Pull request merged**